notes.css

```diff
@@ -0,0 +1,30 @@
+#new-store-form {
+  margin:auto;
+  width: 50%;
+}
+legend, label, input{
```

Comment:
This doesn't really matter, but it's prettier if you put a space before the open curly brace.